terraform/modules/ops/variables.tf

```diff
@@ -33,3 +33,7 @@ variable "setup_cd_system" {
   description = "Create deployment triggers. Enable only if Cloud Build has been granted GitHub access."
 }
 
+variable "pubsub_iam_roles_list" {
```

Comment:
move these into a locals {} block instead of variables. 

We would use variables only for values that we would want an end user to change